ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestmlag.py

```diff
@@ -27,7 +27,7 @@ def test_definition(self) -> dict:
             return None
 
         anta_tests = [
-            {"VerifyMlagStatus": {}},
+            {"VerifyMlagStatus": None},
```

Comment:
What's the purpose of using None here?

Reply:
Consistency. When we dump the catalogs using `save_catalog`, tests without inputs will all have `null`.